code-challenges.test.js

```diff
@@ -1,5 +1,7 @@
 // ASSESSMENT 2: Coding Practical Questions with Jest
 
+const { number } = require("yargs")
```

Comment:
Careful of this! Sometimes jest will add this to your code and it could cause failing tests even though everything is actually working properly. Make sure to comment this out or remove it entirely!